xml/System.Net.Mail/MailAddress.xml

```diff
@@ -725,11 +725,11 @@ The following code example demonstrates sending an email message by using the <x
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="address">To be added.</param>
-        <param name="displayName">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="address">A <see cref="T:System.String" /> that contains an email address.</param>
+        <param name="displayName">A <see cref="T:System.String" /> that contains the display name associated with address. This parameter can be <see langword="null" />.</param>
+        <param name="result">When this method returns, contains the <see cref="T:System.Net.Mail.MailAddress" /> instance if address parsing succeed</param>
+        <summary>Create a new <see cref="T:System.Net.Mail.MailAddress" />. Does not throw an exception if the MailAddress cannot be created.</summary>
+        <returns>A <see cref="T:System.Boolean" /> value that is <see langword="true" /> if the <see cref="T:System.Net.Mail.MailAddress" /> was successfully created; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the <see cref="T:System.Net.Mail.MailAddress" /> was successfully created; otherwise, <see langword="false" />.</returns>
```